### PR TITLE
Update DG - Update Use Cases

### LINKED ISSUE #517: Update DG - Use Cases
* Edit use cases for `login`, `update`, `customer add`, `customer find` to align with the latest behaviour

### PR DESCRIPTION
**Pull request by zhonghan721**
* Remove Sort Customers
* Edit phrasing for extensions
* Add extension on duplicate prefixes for `customer add` and `login`

Closes #517

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1195,12 +1215,24 @@ otherwise)
...
 **Extensions:**
 
-- 1a. Logged-in Owner does not specify the required field(s)
+- 1a. Logged-in Owner does not specify the required field(s).
+
+    - 1a1. CMS displays an error to Logged-in Owner to key in all the fields required to create a customer.
+
+      Use Case ends.
+
+- 1b. Logged-in Owner specifies duplicated fields.
```
Comment (B-enguin) [inline docs/DeveloperGuide.md:1224]: Should this be standardised with the one above?

**Review by B-enguin — COMMENTED** [review]
Some general comments

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zhonghan721) [reply docs/DeveloperGuide.md:1224]: Do you mean Logged-in Owner types duplicated fields from UC02?

**Review by jianyangg — APPROVED** [review]
LGTM

**Pull request merged**